src/autopas/tuning/AutoTuner.h

```diff
@@ -229,12 +229,12 @@ class AutoTuner {
   bool canMeasureEnergy() const;
 
   /**
-   * Notify the tuner about the iteratePairwise runtime of the current configuration.
-   * Keeps track of the fastest runtime so far and triggers early stopping of the sampling phase if the current runtime
-   * is worse than the fastest runtime by more than the maximum allowed slowdown factor.
-   * @param performance The runtime of the current configuration.
+   * Checks whether the current configuration seems to perform so poorly that it shouldn't be resampled further
+   * If the current configuration seems worse than the previosuly best configuration by more than the
+   * earlyStoppingFactor factor, it will not be sampled again. Uses the _estimateRuntimeFromSamples() function to
```

Comment:
```suggestion
   * earlyStoppingFactor factor, it will not be sampled again this tuning phase. Uses the _estimateRuntimeFromSamples() function to
```